client/GameListItem.js

```diff
@@ -0,0 +1,29 @@
+import { Avatar, ListItem, ListItemAvatar, ListItemSecondaryAction, ListItemText } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const GameListItem = ({ game }) => {
+    return (
+        <ListItem>
+            <ListItemAvatar>
+                <Avatar alt={game.name} src={game.thumbnail} />

```

Comment:
Love the round thumbnail but now you are giving me more work for the mobile app! ;)